### ISSUE TITLE
Allow for importing without parsing the contents of the comic file

### ISSUE THREAD

**Issue by mcpierce** [mcpierce/comixed#543]
This feature depends on #544.

When importing a large number of comics, it would be beneficial to postpone cataloging the contents of the comic files in order to speed up the import process.

To do this, an additional option on the import panel would be to allow importing the selected comics but to only create a record in the comics table for the file and to mark the comic as needing to still be updated.

**Comment by mcpierce** [comment]
This was completed as a part of #544.
